gratipay/models/team/__init__.py

```diff
@@ -367,3 +379,35 @@ def load_image(self, size):
             with self.db.get_connection() as c:
                 image = c.lobject(oid, mode='rb').read()
         return image
+
+
+def cast(path_part, state):
+    """This is an Aspen typecaster. Given a slug and a state dict, raise
+    Response or return Team.
+    """
+    redirect = state['website'].redirect
+    request = state['request']
+    user = state['user']
+    slug = path_part
+    qs = request.line.uri.querystring
+
+    try:
+        team = Team.from_slug(slug)
+    except:
+        raise Response(400, 'bad slug')
+
+    if team is None:
+        # Try to redirect to a Participant.
+        from gratipay.models.participant import Participant # avoid circular import
```

Comment:
Open a new GH issue to refactor this so you don't have to lazy-import. Circular dependencies are a hassle.